START.md

```diff
@@ -115,19 +115,27 @@ module "autospotting" {
   autospotting_min_on_demand_percentage = "50.0"
   autospotting_regions_enabled = "eu*,us*"
 
-  lambda_zipname = "./my-autospotting-build.zip"
-  lambda_runtime = "200"
-  lambda_memory_size = "2048"
-  lambda_timeout = "600"
+  lambda_zipname = "./lambda.zip"
+  lambda_runtime = "python2.7"
+  lambda_memory_size = "256"
+  lambda_timeout = "300"
   lambda_run_frequency = "rate(5 minutes)"
 }
 ```
 
+The `lambda.zip` file can be generated by building it locally using
+`make prepare_upload_data`(look for it in the `build` directory) but you'll need
+to set up your local development environment.
+
+Instead, you can also use the lightest official nightly build available at
```

Comment:
Maybe replace `use` with `download` in order not to encourage specifying it directly?

Reply:
Also did you mean 'latest' rather than 'lightest'? :)